autoload/fzf/vim.vim

```diff
@@ -28,22 +28,54 @@ set cpo&vim
 " Common
 " ------------------------------------------------------------------
 
+function! s:get_if_linux_like_bash()
```

Comment:
Can there be a better way to determine the property of the given bash? For example, I wonder if there's a command that we can run to see the behavior of the bash.

Reply:
Something like `bash -c 'ls /mnt/*'` maybe?

Reply:
> Something like `bash -c 'ls /mnt/*'` maybe? (and see if it returns one of `/mnt/[A-Z]`)

This is a good idea, it seems to avoid the maintenance of new and different types of bash in the future.